Gems/Maestro/Code/Source/Cinematics/AnimAZEntityNode.cpp

```diff
@@ -19,198 +19,190 @@
 #include <Maestro/Types/AnimParamType.h>
 #include <Maestro/Types/AnimNodeType.h>
 
-//////////////////////////////////////////////////////////////////////////
-CAnimAzEntityNode::CAnimAzEntityNode(const int id)
-    : CAnimNode(id, AnimNodeType::AzEntity)
+namespace Maestro
 {
-    SetFlags(GetFlags() | eAnimNodeFlags_CanChangeName);
-}
 
-//////////////////////////////////////////////////////////////////////////
-CAnimAzEntityNode::CAnimAzEntityNode()
-    : CAnimAzEntityNode(0)
-{
-}
+    CAnimAzEntityNode::CAnimAzEntityNode(const int id)
+        : CAnimNode(id, AnimNodeType::AzEntity)
+    {
+        SetFlags(GetFlags() | eAnimNodeFlags_CanChangeName);
+    }
 
-//////////////////////////////////////////////////////////////////////////
-CAnimAzEntityNode::~CAnimAzEntityNode()
-{
-}
+    CAnimAzEntityNode::CAnimAzEntityNode()
+        : CAnimAzEntityNode(0)
+    {
+    }
 
-//////////////////////////////////////////////////////////////////////////
-/// @deprecated Serialization for Sequence data in Component Entity Sequences now occurs through AZ::SerializeContext and the Sequence Component
-void CAnimAzEntityNode::Serialize(XmlNodeRef& xmlNode, bool bLoading, bool bLoadEmptyTracks)
-{
-    CAnimNode::Serialize(xmlNode, bLoading, bLoadEmptyTracks);
-    if (bLoading)
+    CAnimAzEntityNode::~CAnimAzEntityNode()
     {
-        AZ::u64 id64;
-        if (xmlNode->getAttr("AnimatedEntityId", id64))
-        {
-            m_entityId = AZ::EntityId(id64);
-        }
     }
-    else
+
+    /// @deprecated Serialization for Sequence data in Component Entity Sequences now occurs through AZ::SerializeContext and the Sequence
+    /// Component
+    void CAnimAzEntityNode::Serialize(XmlNodeRef& xmlNode, bool bLoading, bool bLoadEmptyTracks)
```

Comment:
as it is marked as deprecated I wonder if its still used, might be worthwile to remove the code and see if it compiles

Reply:
As above, I'm all for doing additional clean up here but I think the point of this one was to modernize they style, and not to risk *any* change.